### DISCUSSION TITLE
Time slot booking

### DISCUSSION THREAD

**Discussion by amandasystems** [lukevella/rallly#139] [Ideas]
They would have two rules:
1. Only one slot per participant (this is a soft rule since Rallly doesn't have authentication)
2. Only one participant per slot

I think this is best implemented as a mode rather than as a separate option for a poll to highlight that it's essentially a different kind of poll, but I know for example Doodle had it as two separate settings when creating a poll (and has since removed both options).

I am willing to have a go at implementing this if I have the time, but I cannot promise to do it yet. I am a PhD student at a university and often have the need to book appointments with undergraduates in my teaching, and there is currently no good tool for this available.

**Comment by lukevella** [comment]
Thanks these are both in the backlog. I do plan to add some form of authentication soon to enable limiting the number of votes for each participant.

**Reply by lekr74** [reply]
Hello,
Sorry to dig it, but I wanted to create a new discussion and github proposed me this one.
I would really love to have a one participant per slot option, as I did deploy Rallly for my little girl's school, and we had two people taking the same slot for one to one meeting with the teacher.
thanks
